app/carbon-fields/cf-post-options.php

```diff
@@ -361,22 +377,24 @@ function crb_attach_related_pages()
 			Field
 				::make('checkbox', 'related_image', __('Related Pages Image', 'alps'))
 				->set_option_value('true')
-				->set_help_text(__('Select to display the feature image for the related pages.', 'alps')),
-			Field
-				::make('checkbox', 'related_image_round', __('Related Pages Round Image', 'alps'))
-				->set_option_value('true')
-				->set_help_text(__('Select to make the featured image round.', 'alps'))
-				->set_conditional_logic([
-					'relation' => 'AND',
-					[
-						'field' => 'related_image',
-						'value' => true,
-					],
-					[
-						'field' => 'related_grid',
-						'value' => false,
-					]
-				]),
+				->set_help_text(__('Select to display the feature image for the related pages.', 'alps'))
+                ->set_width(50),
+            Field
+                ::make('radio', 'related_image_crop', __('Related Page Image Cropping', 'alps'))
+                ->add_options([
+                    'square' => __('Square', 'alps'),
+                    'landscape' => __('Landscape', 'alps'),
+                    'portrait' => __('Portrait', 'alps'),
+                    'circle' => __('Circle', 'alps'),
+                ])
+                ->set_default_value('landscape')
```

Comment:
@YauheniKapliarchuk This needs to be changed to

`->set_default_value(setDefaultRelatedImageCrop())`

I forgot to reset it after testing. I can't push a new commit after your changes, can you make this change please?

Reply:
Okay, sure

Reply:
@claytonk please verify it

Thanks,
Best Regards!

Reply:
Looks good. Thanks for fixing that.